Automattic/woocommerce-payments
# Ensure order attribution inputs are not removed when shortcode cart is updated

Fixes #9001 

#### Changes proposed in this Pull Request
This ensures the `wc-order-attribution-inputs` element isn't removed from the page when the shortcode cart is updated. It's this element that provides the data to populate the Origin field inside WooCommerce orders. This change will solve the problem of the Origin being reported as Unknown on orders placed after the cart has been refreshed.









#### Testing instructions





1. Enable the "Enable Cart-Token implementation for PRBs" flag in your dev tools
2. Add a product to the cart
3. Visit the cart page
4. Update the item quantity and click update cart
5. Place an order via the PRB
6. Navigate to WooCommerce > Orders and check that the Origin field is not `unknown`

-------------------


**Post merge**
